### DISCUSSION TITLE
Table attribute for a Enum propertry

### DISCUSSION THREAD

**Discussion by wgrs** [supabase-community/supabase-csharp#139] [General]
How do I attribute a enum property in a table model? I have tried [Column("abc") as well as [Reference(typeof(abc))] and both throw errors.

**Comment by acupofjose** [comment]
There’s an example of how you can specify an enum in the test suite’s models!

You can find it here:
https://github.com/supabase-community/postgrest-csharp/blob/master/PostgrestTests/Models/Todo.cs
